packages/cli/src/commands/setup/deploy/deploy.js

```diff
@@ -50,6 +50,21 @@ export const builder = (yargs) => {
       description: 'Overwrite existing configuration',
       type: 'boolean',
     })
+    .option('database', {
+      alias: 'd',
+      choices: ['none', 'postgres', 'sqlite'],
+      description: 'Database deployment for Render only',
+      type: 'string',
+    })
+    .check((argv) => {
+      if (argv.provider !== 'render' && argv.database !== undefined) {
+        throw new Error('Database option only available for Render deployment')
+      }
+      if (argv.provider === 'render' && argv.database === undefined) {
+        argv.database = 'postgres'
```

Comment:
```suggestion
        argv.database = 'postgresql'
```

see above re: Syntax for Prisma is `postgresql`.